monkey/tests/unit_tests/monkey_island/cc/resources/auth/test_register.py

```diff
@@ -49,6 +43,23 @@ def test_register_successful(monkeypatch, make_registration_request, mock_authen
     mock_authentication_service.reset_repository_encryptor.assert_called_once()
 
 
+@pytest.mark.parametrize(
+    "register_response", [1111, "adfasdf", None, True, MagicMock(side_effect=Exception)]
```

Comment:
Other options include a dict, and lists/iterables of strings/bytes. Same thing applies to `test_login`